user-guide/mkdocs.yml

```diff
@@ -89,6 +89,7 @@ nav:
     - GeoHazard: usecases/geohazardusecases.md
     - Seismic: usecases/seismicusecases.md
     - Wind and Storm Surge: usecases/windstormsurgeusecases.md
+    - Work in Progress: usecases/workinprogress.md
```

Comment:
Generally, a "Work in Progress" in not something deployed. A merge to `main` deploys this. If your intent is to test, is your test complete? (It worked for me.)

Reply:
@silviamazzoni and I chatted. WIP can be removed, yet pages are still accessible at URL:

| delete | `/user-guide/usecases/kareem/usecase` | `/user-guide/usecases/kareem/usecase2` | `/user-guide/usecases/kareem/usecase3` |
| - | - | - | - |
| <img width="1200" alt="deleted wip test" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/34dfb6a1-1e30-43f7-ab89-b04fb96fa5fe"> | <img width="1200" alt="usecase" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/bece9398-5dbd-46dd-9c6b-be94ed83be23"> | <img width="1200" alt="usecase2" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/10144955-dea9-48c9-9f67-002b980fe94e"> | <img width="1200" alt="usecase3" src="https://github.com/DesignSafe-CI/DS-User-Guide/assets/62723358/5df21c25-15ce-4f7b-a6b2-4dedd3d24040"> |